wp-content/themes/bpr2018/resources/assets/sass/_single_post.scss

```diff
@@ -13,6 +13,7 @@
     p, figcaption {
       a {
```

Comment:
This works great for hyperlinks, but it would be great if we could also fix extra-long non-hyperlinks.